### PR TITLE
fix(ui5-popup): Synchronous opening of a popup from another popup with Enter key now works

### LINKED ISSUE #4861: [SF] List: Pressing "Enter" on a list item opens and immediately closes the dialog its supposed to open
### **Bug Description**
When using an interactive List inside a Popover and pressing "Enter" on a list item, the dialog that the list item was supposed to open is opened and then immediately closed. This does not happen with the "Space" key or a mouse click (the dialog is opened and remains open for the user to see).

### **Expected Behavior**
The dialog that the list item is supposed to open should be opened and should remain open.

### **Steps to Reproduce**
1. Go to https://codesandbox.io/s/list-within-popover-xx6nde?file=/src/App.js
2. Click the "Open Popover" button to open the popover with the list
3. Press "Enter" to select one of the list items
4. The dialog that was supposed to open is opened and then closed.
5. Try again with a mouse key or by pressing "Space"
6. The dialog remains open

### **Isolated Example**
https://codesandbox.io/s/list-within-popover-xx6nde?file=/src/App.js

### **Context**
- UI5 Web Components version: 1.2.1

### PR DESCRIPTION
**Pull request by dimovpetar**
### Scenario:
Popup 1 opens popup 2 when `Enter` key is pressed on a list item (`keydown` event).
```html
<ui5-popover id="popover1">
	Popover 1
	<ui5-list id="list">
		<ui5-li>Open popover 2</ui5-li>
	</ui5-list>
</ui5-popover>
<ui5-popover id="popover2">
	Popover 2
	<ui5-button id="closePopover2">Close popover2</ui5-button>
</ui5-popover>
```

```js
list.addEventListener("ui5-item-click", () => {
    popover1.close();
    popover2.show();
});
```
When popover2 opens it applies its initial focus, which makes the `closePopover2` button the active element. The following `click` event triggered by the `Enter` key is then fired on the new active element, not the list item.
So pressing the list item with Enter ends up as `click` event on the close button.

### The problem without UI5 web components
https://jsbin.com/pewuvaqaki/edit?html,output

### Solution:
Prevent the `click` event by calling `preventDefault` on the `keydown` event inside closed popups.

Fixes #4861

### CODE REVIEW

File: packages/main/test/pages/Popover.html [hunk 1]
```diff
@@ -428,6 +428,21 @@
...
+	<ui5-button id="openChainedPopover1">Open chained popover 1</ui5-button>
+	<ui5-popover id="chainedPopover1">
+		Chained Popover 1
+		<ui5-list id="openChainedPopover2List">
+			<ui5-li>Open chained popover 2</ui5-li>
+		</ui5-list>
+	</ui5-popover>
+	<ui5-popover id="chainedPopover2" placement-type="Bottom">
+		Chained Popover 2
+		<ui5-button id="closeChainedPopover2">Close chained popover2</ui5-button>
```
Comment (BorisDafov) [inline packages/main/test/pages/Popover.html:443]: add a space:
Close chained popover 2

Comment (dimovpetar) [reply packages/main/test/pages/Popover.html]: Done

**Review by BorisDafov — APPROVED** [review]

File: packages/main/test/specs/ResponsivePopover.spec.js [hunk 2]
```diff
@@ -33,19 +33,20 @@ describe("ResponsivePopover general interaction", () => {
...
 		assert.strictEqual(activeElementId, "simpleRPInitialFocus", "Initial focus is not prevented");
 	});
 
-	it("Initial focus not prevented", async () => {
+	it("Initial focus prevented", async () => {
 		const btnOpenPopover = await browser.$("#btnInitialFocusPrevented");
 		await btnOpenPopover.click();
 
 		const activeElementId = await browser.$(await browser.getActiveElement()).getAttribute("id");
+		// await browser.pause(30000)
```
Comment (gmkv) [inline packages/main/test/specs/ResponsivePopover.spec.js:49]: remove this comment

**Review by gmkv — CHANGES_REQUESTED** [review]
LGTM, there is just one small thing

File: packages/main/test/specs/ResponsivePopover.spec.js (hunk 2 above)
Comment (dimovpetar) [reply packages/main/test/specs/ResponsivePopover.spec.js]: Done

**Review by gmkv — APPROVED** [review]

**Pull request merged**